src/main/rx/rx.c

```diff
@@ -108,6 +111,16 @@ uint32_t rcInvalidPulsPeriod[MAX_SUPPORTED_RC_CHANNEL_COUNT];
 #define SKIP_RC_ON_SUSPEND_PERIOD 1500000           // 1.5 second period in usec (call frequency independent)
 #define SKIP_RC_SAMPLES_ON_RESUME  2                // flush 2 samples to drop wrong measurements (timing independent)
 
+#define updateSamples(value, samples, sampleIndex, sampleSize, sampleSum, _type) \
+    static _type samples[sampleSize];                                            \
```

Comment:
This should not be a define. Also, having variable definitions in a define is very bad style - this will implicitly make it break whenever it is used multiple times inside the same scope.